tensorboard/plugins/npmi/npmi_demo.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Sample data exhibiting npmi values."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+from absl import app
+import tensorflow as tf
+import numpy as np
+from tensorboard.plugins.npmi import summary
+
+tf.compat.v1.enable_eager_execution()
+
+# Directory into which to write tensorboard data.
+LOGDIR = "/tmp/npmi_demo"
+
+
+def setup_run(logdir, verbose, run_name):
+    # Getting a list of dummy annotations from a txt file.
+    f = open("./tensorboard/plugins/npmi/mountains.txt", "r")
```

Comment:
Generally preferable to use a context manager so that closing the file correctly is handled automatically, plus a little more concise.

```python
with open("./tensorboard/plugins/npmi/mountains.txt", "r") as f:
    mountains = f.readlines()

Reply:
Changed to python file, so this is not relevant anymore.